core/src/messages/agent.rs

```diff
@@ -125,13 +126,10 @@ impl BackendStatsMessage {
 
         //REF: https://docs.docker.com/engine/api/v1.41/#tag/Container/operation/ContainerStats
         //cpu
-        let cpu_stats = &stats_message.cpu_stats;
-        let precpu_stats = &stats_message.precpu_stats;
+        let cpu_stats = &cur_stats_message.cpu_stats;
+        let precpu_stats = &prev_stats_message.cpu_stats;
```

Comment:
Let's rename this to avoid confusion with the `precpu` fields that Docker defines.